### ISSUE TITLE
More explicit exception message

### ISSUE THREAD

**Issue by dev590t** [scalapy/scalapy#141]
`me.shadaj.scalapy.py.PythonException` don't give the stacktrace of the exception, the position where the exception is raise

It could be difficult to debug in some case:
```
@ py"""f(a)"""   
me.shadaj.scalapy.py.PythonException: <class 'KeyError'> ('list',)
  me.shadaj.scalapy.interpreter.CPythonInterpreter$.$anonfun$throwErrorIfOccured$1(CPythonInterpreter.scala:310)
  me.shadaj.scalapy.interpreter.Platform$.Zone(Platform.scala:10)
  me.shadaj.scalapy.interpreter.CPythonInterpreter$.throwErrorIfOccured(CPythonInterpreter.scala:296)
  me.shadaj.scalapy.interpreter.CPythonInterpreter$.$anonfun$load$2(CPythonInterpreter.scala:320)
  me.shadaj.scalapy.interpreter.CPythonInterpreter$.withGil(CPythonInterpreter.scala:143)
  me.shadaj.scalapy.interpreter.CPythonInterpreter$.$anonfun$load$1(CPythonInterpreter.scala:318)
  me.shadaj.scalapy.interpreter.Platform$.Zone(Platform.scala:10)
  me.shadaj.scalapy.interpreter.CPythonInterpreter$.load(CPythonInterpreter.scala:316)
  me.shadaj.scalapy.py.package$.eval(package.scala:53)
  me.shadaj.scalapy.py.package$PyQuote$.py$extension(package.scala:82)
  ammonite.$sess.cmd23$.<clinit>(cmd23.sc:1)
```
Exception message in python interpreter with position where the exception is raise
```
$ python ages.py
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/XXX/.local/lib/python3.8/site-packages/pytube/contrib/playlist.py", line 33, in __init__
    self.playlist_id = extract.playlist_id(url)
  File "/home/XXX/.local/lib/python3.8/site-packages/pytube/extract.py", line 188, in playlist_id
    return parse_qs(parsed.query)['list'][0]
KeyError: 'list'

```

**Comment by shadaj** [comment]
Thanks for raising this issue @zhenleibb! Nicer stack traces would definitely be helpful. I think primarily this would require some more logic to step through the trace and convert the elements to Scala values when an exception is detected.

**Comment by shadaj** [comment]
Resolved by #297
